website/content/en/docs/building-operators/golang/project_migration_guide.md

```diff
@@ -50,133 +66,227 @@ multigroup: true
+	}
+
+	// TODO(user): Modify this to be the types you create that are owned by the primary resource
+	// Watch for changes to secondary resource Pods and requeue the owner Memcached
+	err = c.Watch(&source.Kind{Type: &appsv1.Deployment{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &cachev1alpha1.Memcached{},
+	})
+	if err != nil {
+		return err
+	}
+
+	err = c.Watch(&source.Kind{Type: &corev1.Service{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &cachev1alpha1.Memcached{},
+	})
+	if err != nil {
+		return err
+	}
+
+	return ctrl.NewControllerManagedBy(mgr).
+		For(&cachev1alpha1.Memcached{}).
+		Complete(r)
 }
 ```
 
-2. Update the import aliases in the `Reconcile()` function and copy over any remaining helper functions from `pkg/controller/<kind>/<kind>_controller.go` to `api/<version>/<kind>_controller.go`.
+### Setting the RBAC permissions
 
-Refer to [`memcached_controller.go`][kb_memcached_controller] for the example implementation of `Reconcile()` logic which would be present in `api/v1alpha1/memcached_controller.go` of Memcached operator.
+The RBAC permissions now are configured via markers whcih will generate and update the manifest files present in `config/rbac/`. See that on top of the `Reconcile()` method you will find them. For our Memcache example it will be: 
 
-## Generate CRDs
+```go
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=apps,resources=services,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list
+// +kubebuilder:rbac:groups=core,resources=services,verbs=get;list;watch;create;update;patch;delete
+```
 
-Run [`make manifests`][generate_crd] to generate CRD manifests. They would be generated inside the `config/crd/bases` folder.
+However, you might configure additional permissions for your project then, see [RBAC markers][rbac_markers] doc for further information. 
 
-## Configuring your test environment
+**Note** You might would like to check the final result of our [memcached_controller.go][memcached_controller] migrated to the new layout. 
 
-Projects are scaffolded with unit tests that utilize the [envtest](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/envtest)
-library, which requires certain Kubernetes server binaries be present locally.
-Installation instructions can be found [here][env-test-setup].
+## Migrating the Main.go
 
-## Operator Manifests
+If you are using third-party API's in your project for example then, it means that you need to copy your customizations from `cmd/manager/main.go` to `cmd/main.go`. You’ll also need to ensure all of the needed schemes have been registered.
```

Comment:
Unlike Previously here we are using "you" to refer the reader

Reply:
In this case, I do not think that `we` will make sense. Since `we` are not using third-party in the example. However, please feel free to provide any suggestion for this place.